### DISCUSSION TITLE
Welcome to Arduino MIDI Library Discussions!

### DISCUSSION THREAD

**Discussion by franky47** [FortySevenEffects/arduino_midi_library#185] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

We use Discussions for general Q&A about the usage of the MIDI Library. Issues are used for bug reports, triaged feature requests and internal task management.

👉 If you're not sure where to ask your question, Discussions are the right place.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lathoub** [comment]
Great idea to have a discussion section @franky47. Hi everyone, I'm @lathoub and the maintainer of the [BLE](https://github.com/lathoub/Arduino-BLE-MIDI), [USB](https://github.com/lathoub/Arduino-USBMIDI) and [AppleMIDI/RtpMIDI](https://github.com/lathoub/Arduino-AppleMIDI-Library) Transports that are based on this great library.

**Comment by lathoub** [comment]


**Reply by franky47** [reply]
Wow, are there any RAM usage differences when using that method vs a traditional C-style function reference ? Because this is very cool, and much less verbose!

**Reply by franky47** [reply]
That's awesome, I'll update the wiki :+1:

**Comment by lathoub** [comment]


**Reply by franky47** [reply]
We could have a "Tips & Tricks" discussion 🙂

Edit: there it is: https://github.com/FortySevenEffects/arduino_midi_library/discussions/186

**Reply by lathoub** [reply]
deleted my original comment - not sure if this was a good thing...

**Comment by alwaysuptil1** [comment]
New here to all of this.  Trying to find a way to use the MIDI library with other boards such as the ESP32 or the Seeed XIAO? I need more speed than the arduinos offer....
